src/components/SearchBox/__tests__/SearchBox-test.js

```diff
@@ -184,6 +184,37 @@ describe('SearchBox', () => {
   });
 
   describe('Events', () => {
+    describe('focus/blur', () => {
```

Comment:
I'd add a note saying that this test suite is necessary for `queryHook` to work as documented.

Reply:
Does the next test description specify this? 
> - does not derive value from prop when focused
> - derives value from prop when not focused

Reply:
This is a technical definition. I would mention in a comment something along:

```suggestion
    // This test suite is necessary for the `queryHook` option to be synced
    // with InstantSearch.
    // The query coming from the refinements should not take over the query
    // from the state when the input is focused because it would override
    // the query coming from the `queryHook` function.
    describe('focus/blur', () => {
```

Reply:
I think this formulation is vague, and does not explain what's happening: `the queryHook to be synced`, `take over the query from the state`.
Stated more precisely, the `query` is not "coming" from the `queryHook`, the `query` is exposed by the connector.

How about this?

> This makes sure we don't override the user's input while (s)he's typing. This issue is more obvious when using `queryHook` to add a debouncing.

Reply:
Yeah go ahead! If I can't phrase it clearly it means it definitely needs some explanation!

- `(s)he's typing` → `they're typing`

Reply:
ok, done here: 51e21a5eaf865fbad19eae9982fcd7ef2b16e62c